packages/validator/templates/configSecurity.json

```diff
@@ -0,0 +1,98 @@
+{
+  "config": {
+    "ois": {
+      "__arrayItem": {
+        "__oisSpecs": {}
+      }
+    },
+    "triggers": {
+      "__optional": {
+        "request": {
+          "__arrayItem": {
+            "endpointId": {
+              "__regexp": "^0x[a-fA-F0-9]{64}$"
+            },
+            "oisTitle": {},
+            "endpointName": {}
+          }
+        }
+      }
+    },
+    "nodeSettings": {
+      "providerIdShort": {
+        "__regexp": "^[a-fA-F0-9]{7}$"
+      },
+      "nodeVersion": {
+        "__regexp": "^([0-9]*\\.){2}([0-9]*)$"
+      },
+      "cloudProvider": {
+        "__regexp": "^aws$"
+      },
+      "region": {},
+      "stage": {},
+      "logFormat": {
+        "__regexp": "^(json|plain)$"
+      },
+      "chains": {
+        "__arrayItem": {
+          "id": {},
+          "type": {
+            "__regexp": "^evm$"
+          },
+          "providers": {
+            "__arrayItem": {
+              "name": {},
+              "url": {
+                "__regexp": "^(https?|ftp)://[^\\s/$.?#].[^\\s]*$"
+              },
+              "__optional": {
+                "blockHistoryLimit": {},
+                "minConfirmations": {}
+              }
+            }
+          },
+          "__optional": {
+            "providerAdminForRecordCreation": {
+              "__regexp": "^0x[a-fA-F0-9]{40}$"
+            },
+            "contracts": {
+              "__keyRegexp": "^(Airnode|Convenience)$",
+              "__objectItem": {
+                "__regexp": "^0x[a-fA-F0-9]{40}$"
+              }
+            }
+          }
+        }
+      }
+    },
+    "id": {
+      "__conditions": [
+        {
+          "__if": {
+            "__this": ".*"
+          },
+          "__rootThen": {
+            "security": {
+              "id": {
+                "__regexp": "^__match$",
+                "__level": "error"
+              }
+            }
+          }
+        }
+      ]
+    }
+  },
+  "security": {
```

Comment:
There seems to be some discrepancy between this template and the `security.json` spec ... I think.
https://github.com/api3dao/api3-docs/blob/8e41ab2597cb19cb75e4904d6e9b214287c1d738/airnode/security-json.md#securityjson

It seems like `security.json` should have only two keys: `id` and `apiCredentials`.

I'm not sure what `"__keyRegexp": "^(?!id).*$"` refers to ... maybe some outdated spec?

Reply:
Also entries here need to be matched to security schemes of OISes in `config`. We will get rid of `security.json`, but it will be replaced by a similar field in `config` so the work here can be reused.

Reply:
Yes, this seems like some outdated part of template, what it means is basically there must be something that doesn't have `id` as a key and have the structure specified below, which doesn't match with current structure of `security.json`.
I added this to #197, matching with security schemes of OISes is already planned in #197.